src/js/event-target.js

```diff
@@ -138,6 +138,11 @@ EventTarget.prototype.one = function(type, fn) {
 EventTarget.prototype.trigger = function(event) {
   const type = event.type || event;
 
+  // deprecation
+  // In a future version we should default target to `this`
+  // similar to how we default the target to `elem` in
```

Comment:
I am not sure that we can fix `EventTargets` setting `target` to `document` by default in anything but a major version.